mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraAnnotationMapper.java

```diff
@@ -66,6 +66,7 @@ public void deleteAnnotation(String key) {
     }
 
     public void insertAnnotation(MailboxAnnotation mailboxAnnotation) {
+        Preconditions.checkArgument(!mailboxAnnotation.isNil());
```

Comment:
In fact, I don't know to write the testing for this case.
Before, I had the specific testing for this class (CassandraAnnotationMapperTest) but it's require to delete cause of redundance.
I cannot test from MailboxMapper such as cannot call insertAnnotation with a Nil value.
Sorry but could you give me the guide?